packages/trader/src/Stores/Modules/Trading/trade-store.js

```diff
@@ -565,6 +565,13 @@ export default class TradeStore extends BaseStore {
                                 type: response.msg_type,
                                 ...response.error,
                             });
+
+                            if (isMobile() && this.root_store.common?.services_error?.type === 'buy') {
```

Comment:
I assume we add the timeout to wait for the toast to disappear. 
Can you add a comment on why the timeout is needed for our future reference?

Reply:
Yes, you're right 👍🏼